### ISSUE TITLE
Town names in Vermont require "town of" to find

### ISSUE THREAD

**Issue by thomastrillium** [pelias/pelias#713]
Hey team! 

I was using your awesome geocoding engine when I noticed something interesting.
Let me tell you more about it.

---
#### Here's what I did :innocent:
In otp instance at https://plan.govermont.org/ (linked to pelias) tried searching for a town name such as
-Bennington
-Barre
-Rutland

---
#### Here's what I got :scream_cat:
Addresses that have streets named after these towns, but no the towns themselves.

![screen shot 2018-02-26 at 12 48 35 pm](https://user-images.githubusercontent.com/7597236/36696217-698b3ba0-1af8-11e8-947b-5bc0d63c3572.png)

Here's the API call to Geocode.earth 

https://api.geocode.earth/v1/autocomplete?api_key=________&text=bennington&size=10&focus.point.lat=44.1349&focus.point.lon=-72.8837

---
#### Here's what I was expecting :sparkles:
Generally, the user expects a town name to first show the town, then other instances using the word.

The desired result is achievable by adding "town of" before any of the town names in the search.

---

**Comment by orangejulius** [comment]
After some investigation it looks like this is another case of one of our longest standing autocomplete issues (https://github.com/pelias/pelias/issues/330). Basically, autocomplete when paired with a focus point returns almost exclusively addresses and venues, and rarely cities or other administrative areas.

[Bennington](https://spelunker.whosonfirst.org/id/101728409/) is included in Who's on First data and has good population information.

Here's the query shown above in the [compare tool](http://pelias.github.io/compare):

[/v1/autocomplete?focus.point.lat=44.1349&focus.point.lon=-72.8837&text=bennington](http://pelias.github.io/compare/#/v1/autocomplete%3Ffocus.point.lat=44.1349&focus.point.lon=-72.8837&text=bennington)

![image](https://user-images.githubusercontent.com/111716/36698923-3869827c-1b19-11e8-8e97-3ca501129edc.png)

We've recently added autocomplete support for the [Placeholder](https://github.com/pelias/placeholder) service, which handles administrative areas, but haven't incorporated that into the API results for autocomplete yet. That will likely help. We haven't figured out when these improvements will be made in our new post-Mapzen world, but it's on our radar as part of the [Autocomplete V2 Milestone](https://github.com/pelias/pelias/milestone/3).

**Comment by thomastrillium** [comment]
OK that makes sense. Do you have a general timeline estimate on incorporation of Placeholder? (as in, are we talking weeks or months?)

**Comment by orangejulius** [comment]
It's likely to be a month or more. One workaround would be to avoid specifying a focus point. Pelias can be configured to exclude results outside of a bounding box, which I think would work well enough for Vermont.

**Comment by drewda** [comment]
@orangejulius thanks for the suggestion of using `boundary.rect` with `/v1/autocomplete` -- it's a solid improvement:

![2018-02-27_10-05-38](https://user-images.githubusercontent.com/212369/36746000-0b3601b2-1ba6-11e8-92a0-064dfa11cffe.png)

**Comment by orangejulius** [comment]
That's great!

**Comment by orangejulius** [comment]
I think this is fixed. The query I linked previously now shows both the Geonames 'Town of Bennington' record and a WOF record 'Bennington':
<img width="354" alt="Screen Shot 2019-05-13 at 7 24 00 AM" src="https://user-images.githubusercontent.com/111716/57617992-2af45580-7550-11e9-86f2-fce3964de489.png">

I suspect this came both from general improvements we've made in the past year to Pelias queries, as well as the WOF data import which brought in many many localities from Geonames.

Let me know if there are other searches that still don't work.
